examples/spring/README.md

```diff
@@ -7,3 +7,12 @@ A Spring Boot example application to show how to use Bugsnag in a Spring based J
     ```shell
     ../../gradlew clean bootRun
     ```
+
+## Running in Docker
+
+Execute the following commands in the root of the repository:
+
+```
+docker-compose build
+docker-compose up spring
```

Comment:
this application never seems to end during my testing and has to be killed manually, is this expected?